chamber/models/__init__.py

```diff
@@ -276,10 +280,20 @@ class SmartModel(AuditModel, metaclass=SmartModelBase):
     dispatchers = []
 
     def __init__(self, *args, **kwargs):
-        super(SmartModel, self).__init__(*args, **kwargs)
+        super().__init__(*args, **kwargs)
+        self.is_adding = True
```

Comment:
Možná než is_adding bych použil is_creating? vůbec nejlepší by asi bylo being_created/being_changed, ale to už asi zní divně :(

Reply:
já se řídíl podle djanga oni také mají _state.add. Tak jsem to chtěl udělat podobné

Reply:
ok

Reply:
v kódu je to pak čitelnější, když používáš is_changing a is_adding. Ano je to aby to nekorespondovalo s djangem navíc je dobré aby to nebylo protected